app/javascript/mastodon/features/ui/components/focal_point_modal.js

```diff
@@ -365,8 +365,8 @@ class FocalPointModal extends ImmutablePureComponent {
               <CharacterCounter max={1500} text={detecting ? '' : description} />
```

Comment:
I think the button right above that needs `type='button'`.

Reply:
Good call! 🔥